File: String/Length of Last Word/lengthOfLastWord.c
int lengthOfLastWord(char* s) {
    int i,len = strlen(s);
    i = len-1;
    while(i>=0 && s[i]==' '){
        i--;
    }
    if(i<0)
        return 0;
    int j=i;
    while(j>=0 && s[j]!=' '){
        j--;
    }
    return i-j;
}

File: String/Length of Last Word/lengthOfLastWord.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        if s.count(' ')==len(s):
            return 0
        a = s.split(' ')
        while a[-1]=='':
            del(a[-1])
        return len(a[-1])
        
